api/article/views.py

```diff
@@ -1,4 +1,6 @@
 from rest_framework import viewsets, permissions
+from rest_framework.decorators import detail_route
+
 
 from api.article.serializers import *
```

Comment:
That is a bad way to import smt with *. Better way is `from api.article import serializers`

Reply:
Yes, I forgot to fix it here.